src/pages/Main/Main.jsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+function Main() {
```

Comment:
я бы страницам давал бы постфикс Page
```suggestion
function MainPage() {
```